.circleci/config.yml

```diff
@@ -1,27 +1,32 @@
 version: 2.1
-executors:
-  default:
-    docker:
-      - image: circleci/golang:1.14-node
 
-jobs:
-  lint:
-    executor:
-      name: default
-    steps:
-      - checkout
-      - run: make check-style
-
-  test:
-    executor:
-      name: default
-    steps:
-      - checkout
-      - run: make test
+orbs:
+  plugin-ci: mattermost/plugin-ci@0.1.0
 
 workflows:
   version: 2
-  untagged-build:
+  ci:
     jobs:
-      - lint
-      - test
+      - plugin-ci/lint:
+          filters:
+            tags:
+              only: /^v.*/
+      - plugin-ci/coverage:
+          filters:
+            tags:
+              only: /^v.*/
+      - plugin-ci/build:
+          filters:
+            tags:
+              only: /^v.*/
+      - plugin-ci/deploy-release-github:
+          filters:
+            tags:
+              only: /^v.*/
+            branches:
+              ignore: /.*/
+          context: matterbuild-github-token
```

Comment:
This bit "worries" me -- the starter template is meant for endusers to take and use however they see fit, with a minimum of "bits" that have to be removed. Should we have a `mattermost-plugin-internal-starter-template` that uses the orb instead, and a "generic" one for the rest of the community?